### DISCUSSION TITLE
Can budibase build mobile apps?

### DISCUSSION THREAD

**Discussion by supermx1** [Budibase/budibase#1208] [Help] [answered]
Can Budibase build mobile apps? If so can we get a quick tutorial?

**Comment by shogunpurple** [comment, marked as answer]
Hi @emekatoby, I can see you have already raised https://github.com/Budibase/budibase/discussions/1084 around this topic, and the question was answered.

Budibase builds web applications. We have plans to make the apps budibase produces more mobile friendly and responsive in the near term - but no plans for native mobile functionality.
